startf/transform.go

```diff
@@ -327,51 +338,35 @@ func (t *transform) ModuleLoader(thread *starlark.Thread, module string) (dict s
 	return t.moduleLoader(thread, module)
 }
 
-// LoadDataset is a function
+// LoadDataset implements the starlark load_dataset function
```

Comment:
Need an integration test, similar to `TestSaveTransformUsingPrev` in `cmd`, otherwise this functionality will break again.